core/src/main/java/org/mobilitydata/gtfsvalidator/parsing/RowParser.java

```diff
@@ -52,8 +53,9 @@
  */
 public class RowParser {
 
-  public static final boolean REQUIRED = true;
-  public static final boolean OPTIONAL = false;
+  public static final FieldLevelEnum OPTIONAL = FieldLevelEnum.OPTIONAL;
+  public static final FieldLevelEnum RECOMMENDED = FieldLevelEnum.RECOMMENDED;
+  public static final FieldLevelEnum REQUIRED = FieldLevelEnum.REQUIRED;
```

Comment:
You could drop these three lines entirely if you made the suggested change below.

Reply:
I didn't see a suggested change below, but I believe I understand what you're proposing. This does make it a little easier to follow. It doesn't make sense for `RowParser` to have to expose `FieldLevelEnum` values this way. 

Please see my updated commit. 

Thanks!